<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pass_gen</class>
 <widget class="QDialog" name="pass_gen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>486</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Light</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>zPassword - Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#333333;</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>422</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="QTabWidget" name="tbwdgt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>411</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Automatic</string>
     </attribute>
     <widget class="QPushButton" name="exec_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
      </property>
      <property name="text">
       <string>Generate!</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>What should a password contain?</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="lnge">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>44</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;
font-size:16px;</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Password length:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="lowerLetter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Lower case</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="upperLetter">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Upper case</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="numerus">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Digits</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="specialChars">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Special characters</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QLabel" name="lbl_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;n = Digit | B = Upper-case letter&lt;/p&gt;&lt;p&gt;b = Lower-case letter | c = Special character&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
font-size:18px;</string>
      </property>
      <property name="text">
       <string>Passwordstructure:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="strct">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">position: absolute;
	background-color: #71A031;
  height: 25px;
  border-radius:2px;
  border:1px solid rgba(50,50,50,0.5);
  color: #fff;</string>
      </property>
      <property name="text">
       <string>Generate!</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-size:18px;
</string>
    </property>
    <property name="text">
     <string>Lower-case letters:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lowerLetters">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>71</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
    </property>
    <property name="text">
     <string>abcdefghijklmnopqrstuvwxyz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="upperLetters">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>140</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
    </property>
    <property name="text">
     <string>ABCDEFGHIJKLMNOPQRSTUVWXYZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>109</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-size:18px;</string>
    </property>
    <property name="text">
     <string>Upper-case letters:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="digits">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>71</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
    </property>
    <property name="text">
     <string>0123456789</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-size:18px;</string>
    </property>
    <property name="text">
     <string>Digits:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="specialCharacters">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>206</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-width: 0.5px;
border-style: solid;
border-color:#726F6F;
color:white;</string>
    </property>
    <property name="text">
     <string>^!\&quot;$%&amp;/()=?*+-;,.&lt;&gt;'#_{}[]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>174</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-size:18px;</string>
    </property>
    <property name="text">
     <string>Special characters:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_close_gen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
background-color: rgba(255, 255, 255, 0%);
color:#726F6F;
</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
